### ISSUE TITLE
Add chart library to page editor

### ISSUE THREAD

**Issue by JayDaley** [ietf-tools/wagtail_website#236]
Any suitable chart library will do so long as the integration is such that a chart can be added in the same way a paragraph or table is added when editing a page. Don't need the charts to reference any external data. Charts should work well on mobile and support accessibility (though I have no idea how charts are meant to do that).

**Comment by mgax** [comment]
https://github.com/overcastsoftware/wagtailcharts looks like a good candidate. It's based on https://www.chartjs.org, which claims to display responsive charts.

**Comment by ghwood** [comment]
@JayDaley are charts a current requirement?

**Comment by JayDaley** [comment]
Yes, simple charts embedded in blog posts would be very useful.  Don't want anything too complex
